packages/frontend/src/components/ProjectConnection/DbtSettingsForm.tsx

```diff
@@ -53,6 +55,8 @@ const DbtSettingsForm: FC<DbtSettingsFormProps> = ({
     }, [health, type]);
 
     const form = useMemo(() => {
+        reset({ 'dbt.type': type }); // Reset all defaultValues in forms
```

Comment:
Is this working well when editing an existing project ?

Reply:
It looks good to me 

![Peek 2022-05-03 10-24](https://user-images.githubusercontent.com/1983672/166423748-03ae11cd-87bd-40e0-b6b9-f26df519f26d.gif)